package/MDAnalysis/coordinates/XDR.py

```diff
@@ -61,14 +76,46 @@ def read_numpy_offsets(filename):
     -------
     offsets : dict
         dictionary of offsets information
+
     """
     return {k: v for k, v in six.iteritems(np.load(filename))}
 
 
 class XDRBaseReader(base.Reader):
-    """Base class for libmdaxdr file formats xtc and trr"""
+    """Base class for libmdaxdr file formats xtc and trr
+
+    This class handles integration of XDR based formats into MDAnalysis. The
+    XTC and TRR classes only implement `write_next_timestep` and
+    `_frame_to_ts`.
+
+    Notes
```

Comment:
I would like this to show up in the XTC and TRR docs as well but they don't right now.